src/store/movie/movie.actions.ts

```diff
@@ -0,0 +1,46 @@
+import {createAction, props} from '@ngrx/store';
+import {Movie, SORT} from '@model/movies.model';
+
+export const deleteAllMovies = createAction(
+  '[MOVIE] DELETE_ALL_MOVIES'
```

Comment:
o tym rozmawialiśmy, to [MOVIE] jest zbędne jeśli importujemy akcje, a nie bazujemy na jej nazwie